README.md

```diff
@@ -389,6 +389,8 @@ These settings can be changed at any time.
   Bottlerocket sets this value to false by default. 
 * `settings.ecs.loglevel`: The level of verbosity for the ECS agent's logs.
   Supported values are `debug`, `info`, `warn`, `error`, and `crit`, and the default is `info`.
+* `settings.ecs.spot-instance-draining-enabled`: If the instance receives a spot termination notice, the agent will set the instance's state to `DRAINING`, so the workload can be moved gracefully before the instance is removed. Defaults to `false`.
+  Supported values are `debug`, `info`, `warn`, `error`, and `crit`, and the default is `info`.
```

Comment:
I believe this sentence was added by mistake and isn't applicable to this setting. :)

Reply:
sry :) yes